src/__tests__/patients/related-persons/NewRelatedPersonModal.test.tsx

```diff
@@ -108,10 +108,11 @@ describe('New Related Person Modal', () => {
       wrapper.update()
 
       const errorAlert = wrapper.find(Alert)
+
       expect(onSaveSpy).not.toHaveBeenCalled()
       expect(errorAlert).toHaveLength(1)
       expect(errorAlert.prop('message')).toEqual(
-        'patient.relatedPersons.error.relatedPersonRequired patient.relatedPersons.error.relationshipTypeRequired',
+        'patient.relatedPersons.error.relatedPersonErrorBanner',
```

Comment:
now that we have different behaviors when the two fields are not entered, could we break this into two tests: 

- [ ] one test that when related person is not entered, the banner message appears
- [ ] one test that when relationship type is not entered, the banner message appears, the field is marked invalid, and the field has feedback message on it